CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
-## 0.4.2 An updated 'Alpha' release to update dependencies, and improve logging and stability.
-- The majority of functionality is implemented in the [probe-rs-debugger](https://github.com/probe-rs/probe-rs/tree/master/debugger). Please refer to the [probe-rs CHANGELOG](https://github.com/probe-rs/probe-rs/blob/master/CHANGELOG.md) for detailed innformation on changes that affect functionality in this extension.
+## 1.0.0 The first stable release of the extension. 
```

Comment:
I would keep the release number in sync with probe-rs so 0.17.0 atm. Many if not most extensions have 0.x version numbers afaik.

Reply:
I can live with that :)